environment/src/main/java/com/toggl/environment/services/calendar/CalendarService.kt

```diff
@@ -0,0 +1,12 @@
+package com.toggl.environment.services.calendar
+
+import org.threeten.bp.OffsetDateTime
+
+interface CalendarService {
+    fun getAvailableCalendars(): List<Calendar>
+    fun getCalendarEvents(
+        fromStartDate: OffsetDateTime,
+        toEndDate: OffsetDateTime,
+        fromCalendar: Calendar
```

Comment:
You can get events from multiple calendars:
```suggestion
        fromCalendars: List<Calendar>
```